docs/more-info/iOS.mdx

```diff
@@ -1,16 +1,31 @@
 ---
-title: "iOS"
-description: "Our iOS SDK allows you to receive Apple HealthKit data via webhooks"
+title: "Apple Health"
+description: "Our iOS SDK allows you to receive Apple HealthKit data via webhooks."
 ---
 
+## Overview
+
+This is a guide to help you get started with iOS, specifically Apple HealthKit. First and foremost Apple HealthKit is a central repository for health and fitness data only available on iPhone and Apple Watch.
+Where Metriport comes in is we provide an SDK that allows you to be able to receive Apple HealthKit data in the background.
```

Comment:
I'd remove "and Apple Watch" -> you can technically import data from other sources into Apple Health.

Also, should we use the terminology "Apple Health" rather than "Apple HealthKit"? HealthKit is the dev tool, but our cxs are looking to integrate with Health.